src/ingest_validation_tools/directory_validator.py

```diff
@@ -23,7 +23,9 @@ def validate_directory(path, paths_dict, dataset_ignore_globs=[]):
     for triple in walk(path):
         (dir_path, _dir_names, file_names) = triple
         # [1:] removes leading '/', if any.
-        prefix = dir_path.replace(str(path), '')[1:]
+        # '\\' converts MS backslashes to forward slashes.
+        # ... and hope no one has an actual backslash in the filename!
+        prefix = dir_path.replace(str(path), '')[1:].replace('\\', '/')
```

Comment:
Perhaps a check for system type could be used here, got the idea from [here](https://stackoverflow.com/questions/1325581/how-do-i-check-if-im-running-on-windows-in-python):
```python
if os.name == 'nt':
    # '\\' converts MS backslashes to forward slashes.
    # ... and hope no one has an actual backslash in the filename!
    prefix = dir_path.replace(str(path), '')[1:].replace('\\', '/')
else:
    prefix = dir_path.replace(str(path), '')[1:]
```

Reply:
Will do, thanks.